app/metrics.yaml

```diff
@@ -78,7 +78,7 @@ events:
         description: >
           A string containing the name of the item the user tapped. These items include:
           Settings, Library, Help, Desktop Site toggle on/off, Find in Page, New Tab,
-          Private Tab, Share, Report Site Issue, Back/Forward button, Reload Button
+          Private Tab, Share, Report Site Issue, Back/Forward button, Reload Button, Quit
     bugs:
     - 1024
     data_reviews:
```

Comment:
Could we add a link to this PR for this data review?

Reply:
Just did!